framework/core/js/src/common/utils/dayjsPlugins.ts

```diff
@@ -0,0 +1,24 @@
+const customFormats: import('dayjs').PluginFunc = function (_option, c, _factory) {
```

Comment:
And there need some head comments above the function signature.

Need to explain how `f` and `F` work, what they mean, and how to replace.

Because this function contains regex which is hard to understand, so if there are no notes about how this function works, it may be hard to maintain it in the future.

Reply:
> And there need some head comments above the function signature.

It's a DayJS plugin. You can get some info from DayJS docs, I can add `@see`.

> Need to explain how `f` and `F` work, what they mean, and how to replace.

Their meaning is already in the PR's description. And to replace them, you only need to do it like how you did with DayJS's Localized Formats.

> this function contains regex which is hard to understand

It works exactly like DayJS's Localized Format plugin.

Reply:
add it to the code comments. Although know that, the later programmer may not.